tests/Fractions.Tests/FractionSpecs/FromDouble/Method_FromDouble.cs

```diff
@@ -288,4 +288,21 @@ public class When_a_fraction_is_created_with_1_third : Spec {
         var rounded = Math.Round(_fraction.ToDouble(), 15, MidpointRounding.ToEven);
         rounded.Should().Be(0.333333333333333);
     }
+}
+
+[TestFixture]
+public class When_a_fraction_is_created_from_double : Spec {

```

Comment:
Note: this isn't a regression, I'm just highlighting an existing _issue_ (that I personally wasn't aware of).

Reply:
I didn't even try to piece together the double data type at the bit level. In most cases, the contents of a double data type are just an approximation (due to the base to 2), so I personally didn't find it worth the effort.

The `ToDouble()'` method is defined as follows:

```csharp
     /// <summary>
    /// Returns the fraction as (rounded!) floating point value.
    /// </summary>
    /// <returns>A floating point value</returns>
    public double ToDouble() => IsZero ? 0 : (double)Numerator / (double)Denominator;
```

Reply:
Will change the test to:

```csharp
[TestFixture]
public class When_a_fraction_is_created_from_double : Spec {
    private const double VerySmallValue = 1.0 / (double.MaxValue - 100);
    private Fraction _fraction;

    public override void Act() =>
        _fraction = Fraction.FromDouble(VerySmallValue);

    [Test]
    public void ToDouble_does_not_return_the_exactly_the_same_value() =>
        _fraction.ToDouble().Should().NotBe(VerySmallValue);

    [Test]
    public void ToDouble_returns_an_approximation() {
        var approximateValue = _fraction.ToDouble();

        Math.Abs(approximateValue)
            .Should()
            .BeLessThanOrEqualTo(double.Epsilon);
    }
}
```